.idea/compiler.xml

```diff
@@ -6,8 +6,8 @@
         <sourceOutputDir name="target/generated-sources/annotations" />
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
+        <module name="tickbox-integrated-tests" />
```

Comment:
I strongly suggest you remove IDE-specific files from the source repository

Reply:
This is a result of me forgetting to add gitignore on project creation